workflow-templates/docker-build-template.yml

```
@@ -0,0 +1,72 @@
+name: Docker Build and Push
+
+on:
+  push:
+    branches:
+      - 'main'
```

`stable` is the default branch at HG
```suggestion
      - 'stable'
```

---

Updated branch name to stable